packages/core/src/utils/constants.js

```diff
@@ -35,3 +35,11 @@ export const delay = {
 	FAST: 200,
 	SLOW: 600
 };
+
+export const keys = {
```

Comment:
`keys` is too generic and not clear. What do you think about `keyboardEventKeys` ?